python/pyspark/sql/utils.py

```diff
@@ -179,7 +179,7 @@ def is_remote() -> bool:
     return "SPARK_CONNECT_MODE_ENABLED" in os.environ
 
 
-def try_remote_functions(f: FuncT) -> FuncT:
```

Comment:
you could just leave this as is, and fix `functions.py` when you import with an alias. e.g., `from pyspark.sql.utils import try_remote_functions as _try_remote_functions`

Reply:
ok